### ISSUE TITLE
Coordination for working on this project?

### ISSUE THREAD

**Issue by SourceCodeDeleted** [BoomerangDecompiler/boomerang#266]
Not really an issue .

But do you have a discord where this project is being coordinated and worked on? 

This project seems to have a lot of potential.

**Comment by ceeac** [comment]
No, there is not - IMO this is not really necessary at the moment given the amount of activity.
